### PR TITLE
update text for Solidity scan

### LINKED ISSUE #4882: Update Solidity Scan Modal text
The current text needs editing:
To scan the contract for vulnerabilities & possible risks, smart contract code will be shared to third-party SolidityScan.

What are you informing the user? Is it that we are sharing your code?  Or what Solidity Scan does?  
If sharing, I don't think the word share is the right. Its more we are sending it Solidity Scan. 
Does it need to be vulnerabilities and risk.  Is just vulnerabilities is enough?  And if not then the word possible is not needed.

Choices:
A. To scan the contract for vulnerabilities and risks, the smart contract's code will be shared with SolidityScan, a  third-party.

B. Scanning this contract for vulnerabilities and risks, the smart contract code will be analyzed by SolidityScan, a  third-party provider.

C.  SolidityScan, a  third-party provider, will scan this contract for vulnerabilities and risks.

D.  SolidityScan, a  third-party provider, will scan this contract for vulnerabilities.

### PR DESCRIPTION
**Pull request by Aniket-Engg**
fixes #4882 

Added matomo for external links

### CODE REVIEW

**Review by vermouth22 — DISMISSED** [review]
Just change ampersands to "and" 

Also, looks like there is no space between end of sentence and the "Learn More" link

**Comment by Aniket-Engg** [comment]
> Just change ampersands to "and"
> 
> Also, looks like there is no space between end of sentence and the "Learn More" link

I have updated it

**Review by ryestew — DISMISSED** [review]

File: libs/remix-ui/run-tab/src/lib/components/solScanTable.tsx [hunk 1]
```diff
@@ -13,7 +14,13 @@ export function SolScanTable(props: SolScanTableProps) {
   return (
     <>
       <p>Scanning successful! <b>{scanDetails.length} warnings </b> found for file: <b>{fileName}</b></p>
-      <p>See the warning details below. For more details, <a href="https://solidityscan.com/signup" target='blank'>Go to SolidityScan</a></p>
+      <p>See the warning details below. For more details,
```
[the remark this answers is not in the document]
Comment (Aniket-Engg) [reply libs/remix-ui/run-tab/src/lib/components/solScanTable.tsx]: updated

File: libs/remix-ui/run-tab/src/lib/components/solScanTable.tsx [hunk 2]
```diff
@@ -13,7 +14,13 @@ export function SolScanTable(props: SolScanTableProps) {
...
+        <a href="https://solidityscan.com/signup"
+          target='_blank'
+          onClick={() => _paq.push(['trackEvent', 'udapp', 'solidityScan', 'goToSolidityScan'])}>
+            Go to SolidityScan
```
[the remark this answers is not in the document]
Comment (Aniket-Engg) [reply libs/remix-ui/run-tab/src/lib/components/solScanTable.tsx]: updated

**Review by yann300 — APPROVED** [review]

**Pull request merged**